manipulator:
  joints: ['joint1', 'joint2', 'joint3']
  directions: [-1, 1, -1]
  origin: [0, 0, 0.2, 1.57079, 0.0, 0.0]
  home: [-0.443, 0.907, -0.463]
  limits:
    upper: [1.57, 1.57, 2.3]
    lower: [-1.57, -1.57, -2.3]
  lambda: 30.0
UAV:
  degrees_of_freedom: 4
  lambda: 30.0
IMPEDANCE_FILTER:
  M: [2.5, 2.5, 2.5, 0.0833333, 0.0833333, 0.0833333]
  B: [10, 10, 10, 0.5270462767, 0.5270462767, 0.5270462767]
  K: [10, 10, 10, 0.83333333 , 0.83333333, 0.83333333]
  dead_zone: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
AIC:
  INTEGRAL_ADAPTATION_GAINS:
    ke1: [0.4, 0.001, 0.001, 0.0, 0.0, 0.0]
  PROPORTIONAL_ADAPTATION_GAINS:
    ke2: [1.0, 0.018, 0.018, 0.0, 0.0, 0.0]
  INITIAL_GAINS:
    Ke: [0.0, 0.003, 0.003, 1.0, 1.0, 1.0]
  WEIGHTING_FACTORS:
    Wp: [1.5, 1.0, 1.0, 0.0, 0.0, 0.0]
    Wd: [0.8, 0.0, 0.0, 0.0, 0.0, 0.0]